receiver/lokireceiver/loki.go

```diff
@@ -163,8 +163,9 @@ func (r *lokiReceiver) Push(ctx context.Context, pushRequest *push.PushRequest)
 		return &push.PushResponse{}, err
 	}
 	ctx = r.obsrepGRPC.StartLogsOp(ctx)
+	logRecordCount := logs.LogRecordCount()
 	err = r.nextConsumer.ConsumeLogs(ctx, logs)
-	r.obsrepGRPC.EndLogsOp(ctx, "protobuf", logs.LogRecordCount(), err)
+	r.obsrepGRPC.EndLogsOp(ctx, "protobuf", logRecordCount, err)
```

Comment:
I see there's another similar case in the  `handleLogs` function in this file, should that be addressed here as well?